tests/ops/op_math/test_sum.py

```diff
@@ -659,6 +659,45 @@ def test_simplify_method(self):
             assert s1.data == s2.data
             assert s1.arithmetic_depth == s2.arithmetic_depth
 
+    def test_simplify_grouping(self):
```

Comment:
Add a test with the new tolerance parameter?

Reply:
Done!